packages/next/build/webpack-config.ts

```diff
@@ -668,7 +664,7 @@ export default async function getBaseWebpackConfig(
                             'Custom <App>'
                           )}. Please move all global CSS imports to ${chalk.cyan(
                             customAppFile
-                              ? path.relative(dir, customAppFile)
+                              ? path.relative(pagesDir, customAppFile)
```

Comment:
This should still be `dir`.

Reply:
I think if you leave this line as `dir` you will see the path to unused files that are not used.

Reply:
Using `dir` is so the path printed is displayed as:
```
pages/app.ts
// or
src/pages/app.ts
```

When set to `pagesDir`, it'll always emit `app.ts` -- this is not the intended behavior. The path should be where the `app.ts` file is located relative to the project root.

Reply:
You're right. This is my mistake.

Reply:
No worries, thank you very much for sending this fix!